queueUI/mainwindow.py

```diff
@@ -32,6 +33,25 @@ def __init__(self):
         settings.restoreWindowGeometry(self, "mainwindow_geometry")
         print("Settings are saved in:", settings.fileName())
 
+        self.RM = REManagerAPI()
+        self.runEngineOpenButton_3.clicked.connect(self.openRunEngine)
+        self.runEngineCloseButton_3.clicked.connect(self.closeRunEngine)
+
+    # RE Function:
+    def openRunEngine(self):
+        self.RM.environment_open()
```

Comment:
Add a check to avoid attempting to open when env is already open (related to #15).